common/src/main/java/org/apache/uniffle/common/netty/protocol/SendShuffleDataRequest.java

```diff
@@ -103,7 +103,21 @@ private static Map<Integer, List<ShuffleBlockInfo>> decodePartitionData(ByteBuf
       int lengthOfShuffleBlocks = byteBuf.readInt();
       List<ShuffleBlockInfo> shuffleBlockInfoList = Lists.newArrayList();
       for (int j = 0; j < lengthOfShuffleBlocks; j++) {
-        shuffleBlockInfoList.add(Decoders.decodeShuffleBlockInfo(byteBuf));
+        try {
+          shuffleBlockInfoList.add(Decoders.decodeShuffleBlockInfo(byteBuf));
+        } catch (Throwable t) {
+          // An OutOfDirectMemoryError will be thrown, when the direct memory reaches the limit.
+          // OutOfDirectMemoryError will not cause the JVM to exit, but may lead to direct memory
+          // leaks.
+          // Warning: You can refer to docs/server_guide.md to set MAX_DIRECT_MEMORY_SIZE to a
```

Comment:
It's not a warning.
Maybe: Warning: -> Note:

Reply:
Fixed it as you suggested.